packages/perseus-editor/src/components/util.ts

```diff
@@ -52,6 +52,8 @@ export function getValidNumberFromString(value: string) {
     return isNaN(parsed) ? 0 : parsed;
 }
 
+const DEFAULT_COLOR = "green";
```

Comment:
It's the default color for everything?

Reply:
Hmm... good question. I'll double check.